widget/src/overlay/menu.rs

```diff
@@ -526,7 +526,7 @@ where
 
             renderer.fill_text(
                 Text {
-                    content: &option.to_string(),
+                    content: option.to_string(),
```

Comment:
No more `(&option.to_string()).to_string()`! I will sleep tight today.

Reply:
Hey, since you're looking into this `.to_string` reallocations couldn't the `text` helper function avoid reallocation as well here:
[Text::new(text.to_string())](https://github.com/iced-rs/iced/blob/b40db569a2369dbdc79e40dc700974c2f7ac8d4b/widget/src/helpers.rs#L154)
I always tend to use `Text::new(&self.some_str)` instead of `text(&self.some_str)` because of this.

Reply:
Not if we want to keep the `ToString` compatibility (e.g. `text(10)`). Specialization hasn't landed yet in Rust.

Reply:
Didn't think of that. I guess I can always make my own helper! 😄